aiidalab_widgets_base/viewers.py

```diff
@@ -79,6 +80,29 @@ def viewer(obj, downloadable=True, **kwargs):
         raise exc
 
 
+class DisplayAiidaObject(ipw.VBox):
```

Comment:
```suggestion
class AiidaNodeViewWidget(ipw.VBox):
```

Reply:
Do you want to expose the widget in the package root namespace?

Reply:
> Do you want to expose the widget in the package root namespace?

right, will do.